tt_metal/impl/dispatch/device_command.hpp

```diff
@@ -16,16 +16,12 @@ class DeviceCommand {
 
     // Constants
     //TODO: investigate other num_cores
+    static constexpr uint32_t MAX_HUGEPAGE_SIZE = 1 << 30; // 1GB;
```

Comment:
Should this not just be pulled in at runtime from the device/cluster?

Reply:
I wanted to have access to the size of hugepage so I can get the channel offset for device A assigned to channel X without needing the device ids for channels 0 ... X-1. I would need these device ids to get the channel sizes

This assumes each hugepage is the same size (right now they are)